src/scripts/main.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+const gallery = document.querySelector('.gallery');
+
+const changeImage = (eventElement) => {
+  const largePrevious = document.querySelector('.gallery__large-img');
+
+  largePrevious.src = eventElement.target.src;
+  eventElement.preventDefault();
```

Comment:
You're trying to prevent the default action of the event after it has already occurred. The `eventElement.preventDefault()` method should be called before any other action in the event handler. This is a bug that could cause undesired behavior.